openshift-knative-operator/hack/update-deps.sh

```diff
@@ -45,10 +45,8 @@ fi
 go mod tidy
 go mod vendor
 
-rm -rf $(find vendor/ -name 'OWNERS')
-# Remove unit tests & e2e tests.
-rm -rf $(find vendor/ -path '*/pkg/*_test.go')
-rm -rf $(find vendor/ -path '*/e2e/*_test.go')
+# Remove unnecessary files.
+find vendor/ \( -name "OWNERS" -o -name "OWNERS_ALIASES" -o -name "BUILD" -o -name "BUILD.bazel" -o -name "*_test.go" \) -print0 | xargs -0 rm -f
 
 # Add permission for shell scripts
-chmod +x $(find vendor -type f -name '*.sh')
\ No newline at end of file
+chmod +x $(find vendor -type f -name '*.sh')
```

Comment:
This will not fail on large number of files

```suggestion
find vendor -type f -name '*.sh' -exec chmod +x {} +
```